qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -277,57 +288,61 @@ class NetworkBaselineTest extends BaseSpecification {
             [])
     }
 
-    @Unroll
-    // TODO: ROX-11126
-    @Ignore
     @Category(NetworkBaseline)
     def "Verify user get for non-existent baseline"() {
         when:
-        "Create initial set of deployments, wait for baseline to populate"
+        "Create initial set of deployments, wait for them to deploy"
         def beforeDeploymentCreate = System.currentTimeSeconds()
-        batchCreate([USER_DEP, BASELINED_USER_CLIENT_DEP])
+        batchCreate([USER_DEP])
         def justAfterDeploymentCreate = System.currentTimeSeconds()
 
         def serverDeploymentID = USER_DEP.deploymentUid
         assert serverDeploymentID != null
 
-        def baselinedClientDeploymentID = BASELINED_USER_CLIENT_DEP.deploymentUid
-        assert baselinedClientDeploymentID != null
-
-        log.info "Deployment IDs Server: ${serverDeploymentID}, " +
-                    "Baselined client: ${baselinedClientDeploymentID}"
+        log.info "Deployment IDs Server: ${serverDeploymentID}"
 
+        // Get the server baseline to simulate a user asking for a baseline prior to
+        // observation ending.
         def serverBaseline = NetworkBaselineService.getNetworkBaseline(serverDeploymentID)
         log.info "Requested Baseline: ${serverBaseline}"
         assert serverBaseline
 
+        // Add a client deployment
+        def beforeClientDeploymentCreate = System.currentTimeSeconds()
+        batchCreate([BASELINED_USER_CLIENT_DEP])
+        def justAfterClientDeploymentCreate = System.currentTimeSeconds()
+
+        def baselinedClientDeploymentID = BASELINED_USER_CLIENT_DEP.deploymentUid
+        assert baselinedClientDeploymentID != null
+        log.info "Client deployment: ${baselinedClientDeploymentID}"
+
+        def clientDeployment = DEPLOYMENTS.find { it.name == BASELINED_USER_CLIENT_DEP_NAME }
+
+        assert retryUntilTrue({
+            // Trigger http traffic to tests service deployment to ensure
+            // StackRox would see it and we can later include it in the baseline.
+            orchestrator.execInContainer(clientDeployment,
+                    "curl --insecure --head --request GET --max-time 5 http://${USER_DEP_NAME}")
```

Comment:
Why do `curl` here if the client calls `wget` anyway? What would go wrong if you remove this line?